content/en/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -155,7 +155,7 @@ runtime where possible.
 
 Another thing to look out for is anything expecting to run for system maintenance
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/reference/tools/map-crictl-dockercli/)) and for the
```

Comment:
```suggestion
former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from dockercli to crictl](/docs/reference/tools/map-crictl-dockercli/)) and for the
```

Bear in mind that this is a fix to the _superseded_ FAQ. I'm happy to LGTM this one given how prominent the dockershim removal is, but would prefer to fix the URL to be relative at the same time.